src/shared/sandboxRequest.ts

```diff
@@ -21,7 +21,7 @@ export const generateSboxName = async (): Promise<string> => {
 };
 
 export async function createSandboxRequest(
```

Comment:
I don't think this overload type change is valid.  If isClone isn't true, would you get back a srcSandboxName like it says ?